### PR TITLE
build: update ng-dev configuration for release

### PR DESCRIPTION
**Pull request by alan-agius4**
Update the configuration for releasing via ng-dev.

```
yarn ng-dev release publish
```

### CODE REVIEW

**Comment by alan-agius4** [comment]
@dgp1130 / @josephperrott can someone please help to document the entire process in https://github.com/angular/universal/blob/master/docs/process/release.md?

**Review by dgp1130 — APPROVED** [review]
Awesome, thanks so much!

File: .ng-dev/release.ts [hunk 1]
```diff
@@ -15,6 +16,10 @@ export const release: ReleaseConfig = {
   },
   publishRegistry: 'https://wombat-dressing-room.appspot.com',
   releasePrLabels: ['action: merge'],
-  // TODO: Set up package building.
-  buildPackages: async () => [],
+  buildPackages: () => {
+    // The buildTargetPackages function is loaded at runtime as the loading the script causes an
+    // invocation of bazel.
+    const { buildTargetPackages } = require(join(__dirname, '../scripts/package-builder.js'));
+    return buildTargetPackages('dist/release-output', false, 'Release', /* isRelease */ true);
```
Comment (josephperrott) [inline .ng-dev/release.ts:23]: I believe there is an extra param here, I think this was copied over from framework where there was an Ivy flag involved as well.

```suggestion
    return buildTargetPackages('dist/release-output', 'Release', /* isRelease */ true);
```

**Review by josephperrott — APPROVED** [review]
LGTM

Just the one comment to address

**Comment by dgp1130** [comment]
Docs follow up in https://github.com/angular/universal/pull/2450.

**Pull request merged**